controllers/tenant/metrics.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020-2025 Project Capsule Authors
+// SPDX-License-Identifier: Apache-2.0
+package tenant
+
+import capsulev1beta2 "github.com/projectcapsule/capsule/api/v1beta2"
+
+// Exposing Status Metrics for tenant.
+func (r *Manager) syncStatusMetrics(tenant *capsulev1beta2.Tenant) {
+	var cordoned float64 = 0
+	// Reset all metrics
+	r.Metrics.DeleteTenantStatusMetrics(tenant.GetName())
+	// Expose namespace-tenant relationship
+	for _, ns := range tenant.Status.Namespaces {
+		r.Metrics.TenantNamespaceRelationshipGauge.WithLabelValues(tenant.GetName(), ns).Set(1)
+	}
```

Comment:
Can you think of a smoother implementation than always deleting all of the exemplers? Even if so, `DeletePartialMatch` is very resource intensive. We should avoid doing that on every loop for scalability. However you might be able to preserve the status namespaces before reconciling and diffing between the namespaces and remove those, which are no longer in the status